config/webpack/terra-dev-site.webpack.config.js

```diff
@@ -57,7 +58,7 @@ const devSiteConfig = (env = {}, argv = {}) => {
   const siteConfig = loadSiteConfig();
 
   // Generate the files need to spin up the site.
-  generateAppConfig(siteConfig, production, verbose);
+  generateAppConfig(siteConfig, production, verbose, includeTestEvidence);
```

Comment:
Can we open up the site configuration to accept `includeTestEvidence: true || false`? Or when are we defining ` generateTestEvidenceConfig ` value?

Reply:
I'd be fine with either, but since we don't have a good way for orion-toolkit-js to inject the setting through config, we opted for the webpack definition. An example of orion-toolkit-js's inject would be:
```
const defaultWebpackConfig = require('terra-toolkit/config/webpack/webpack.config');

const mergedConfig = (env, argv) => {
  if (env && env.generateScreenshotConfig) {
    return defaultWebpackConfig(env, argv);
  }
  return defaultWebpackConfig(Object.assign(env || {}, { generateScreenshotConfig: true }), argv);
};

module.exports = mergedConfig;
```

Reply:
Offline discussion, we are going to use site config and make it the default.